Containerfiles/HeatRXT-Containerfile

```diff
@@ -0,0 +1,15 @@
+ARG VERSION=master-ubuntu_jammy
+
+FROM openstackhelm/heat:${VERSION} as build
+
+RUN apt-get update && apt-get install -y git && apt clean
+
+RUN /var/lib/openstack/bin/pip install git+https://github.com/rackerlabs/heat-rxt.git@stable/2024.1
```

Comment:
@sowm9802  - Ideally we would pull down and install the desired tag/branch directly from `https://opendev.org/openstack/heat` in lieu of a rackerlabs fork, then apply any missing patches to that so that we're not maintaining a separate fork and having it be the source of truth. Maintaining a separate fork may prove cumbersome over time.

Reply:
@LukeRepko Made the required changes, could you please review. Thank you !